src/profiler_caliper.f90

```diff
@@ -127,9 +124,9 @@ subroutine manager_error(mmm)
       logical :: ret
       character(len=:), allocatable :: errmsg
 
-      ret = mgr%error()
+      ret = mmm%error()
```

Comment:
Why not call `mmm` -> `mgr`? It would be easier to read I think.

Reply:
Or if concerned `mgr` might clash with the module-level variable, another name than `mmm` would be prefereable

Reply:
Sounds good, I will use `manager`

Reply:
Fixed